File: copy-assets.js
const fs = require('fs');
const path = require('path');

// Define source and destination paths
const originalPath = path.resolve('d:', 'Github_personal', 'elmentor-landing-page');
const cleanPath = path.resolve('d:', 'Github_personal', 'elmentor-landing-page-clean');

// Asset mappings: [source, destination]
const assetMappings = [
  // Logo files
  [
    path.join(originalPath, 'src', 'assets', 'images', 'elmentor-logo.png'),
    path.join(cleanPath, 'src', 'assets', 'images', 'elmentor-logo.png')
  ],
  // Founder image
  [
    path.join(originalPath, 'src', 'assets', 'images', 'mohamed-radwan.png'),
    path.join(cleanPath, 'src', 'assets', 'images', 'mohamed-radwan.png')
  ],
  // Circles diagram (from public directory)
  [
    path.join(originalPath, 'public', 'elmentor-circles.png'),
    path.join(cleanPath, 'public', 'elmentor-circles.png')
  ],
  // DevOps Visions logos
  [
    path.join(originalPath, 'src', 'assets', 'images', 'DevOps-Visions.png'),
    path.join(cleanPath, 'src', 'assets', 'images', 'DevOps-Visions.png')
  ],
  [
    path.join(originalPath, 'src', 'assets', 'images', 'DevOps-Visions_Horizontal_Green.png'),
    path.join(cleanPath, 'src', 'assets', 'images', 'DevOps-Visions_Horizontal_Green.png')
  ]
];

function copyAssets() {
  console.log('🚀 Starting asset copy process...\n');

  let successCount = 0;
  let errorCount = 0;

  assetMappings.forEach(([source, destination], index) => {
    try {
      // Check if source file exists
      if (!fs.existsSync(source)) {
        console.log(`❌ Source file not found: ${source}`);
        errorCount++;
        return;
      }

      // Ensure destination directory exists
      const destDir = path.dirname(destination);
      if (!fs.existsSync(destDir)) {
        fs.mkdirSync(destDir, { recursive: true });
        console.log(`📁 Created directory: ${destDir}`);
      }

      // Copy the file
      fs.copyFileSync(source, destination);
      console.log(`✅ Copied: ${path.basename(source)}`);
      successCount++;

    } catch (error) {
      console.log(`❌ Error copying ${path.basename(source)}: ${error.message}`);
      errorCount++;
    }
  });

  console.log(`\n📊 Copy Results:`);
  console.log(`✅ Successful: ${successCount} files`);
  console.log(`❌ Failed: ${errorCount} files`);

  if (successCount > 0) {
    console.log(`\n🎉 Asset copying completed! You can now run 'npm run dev' to see the website with all assets.`);
  }
}

// Run the copy process
copyAssets();

File: copy_assets.py
import os
import shutil
from pathlib import Path

def copy_assets():
    # Define paths
    original_base = Path(r"d:\Github_personal\elmentor-landing-page")
    clean_base = Path(r"d:\Github_personal\elmentor-landing-page-clean")
    
    # Asset copy mappings: (source_path, dest_path)
    assets_to_copy = [
        # Logo files
        (original_base / "src/assets/images/elmentor-logo.png", 
         clean_base / "src/assets/images/elmentor-logo.png"),
        
        # Founder image
        (original_base / "src/assets/images/mohamed-radwan.png", 
         clean_base / "src/assets/images/mohamed-radwan.png"),
        
        # Circles diagram
        (original_base / "public/elmentor-circles.png", 
         clean_base / "public/elmentor-circles.png"),
        
        # DevOps Visions logos
        (original_base / "src/assets/images/DevOps-Visions.png", 
         clean_base / "src/assets/images/DevOps-Visions.png"),
        
        (original_base / "src/assets/images/DevOps-Visions_Horizontal_Green.png", 
         clean_base / "src/assets/images/DevOps-Visions_Horizontal_Green.png"),
    ]
    
    print("🚀 Starting asset copy process...")
    success_count = 0
    error_count = 0
    
    for source, dest in assets_to_copy:
        try:
            if source.exists():
                # Create destination directory if it doesn't exist
                dest.parent.mkdir(parents=True, exist_ok=True)
                
                # Copy the file
                shutil.copy2(source, dest)
                print(f"✅ Copied: {source.name}")
                success_count += 1
            else:
                print(f"❌ Source not found: {source}")
                error_count += 1
                
        except Exception as e:
            print(f"❌ Error copying {source.name}: {str(e)}")
            error_count += 1
    
    print(f"\n📊 Results: ✅ {success_count} copied, ❌ {error_count} failed")
    
    if success_count > 0:
        print("🎉 Assets copied successfully! Website should now display all images.")

if __name__ == "__main__":
    copy_assets()
